src/core/src/pass/serialize.cpp

```diff
@@ -1208,6 +1208,14 @@ void serializeFunc(std::ostream& xml_file,
 namespace ov {
 bool pass::Serialize::run_on_model(const std::shared_ptr<ov::Model>& model) {
     RUN_ON_FUNCTION_SCOPE(Serialize);
+
+    // TODO xxx-105807: if rt_info is set in python api as a string ['disable_fp16_compression_0'] = '',
```

Comment:
Name of attribute is different.

Reply:
`_0` looks weird, can we avoid it?

Reply:
I will update the name. But unfortunately we can not get rid of `_0`

Reply:
updated the name